theano/gpuarray/opt.py

```diff
@@ -299,37 +293,33 @@ def apply(self, fgraph):
             if isinstance(i, theano.Constant):
                 mapping[i] = i
         for node in fgraph.toposort():
-            for lopt in (self.local_optimizers_all + 
+            for lopt in (self.local_optimizers_all +
                          self.local_optimizers_map.get(type(node.op), []) +
                          self.local_optimizers_map.get(node.op, [])):
```

Comment:
This change don't need to be here, but having the same order here and bellow would be good.